### DISCUSSION TITLE
I want to use din midi and touch screen at the same time on stm32f411blackpill

### DISCUSSION THREAD

**Discussion by js-midi** [shanteacontrols/OpenDeck#110] [Q&A]
I want to use din midi and touch screen at the same time on stm32f411blackpill
Is it possible?
Please help me~!!

**Comment by paradajz** [comment]
Please stop treating me as your support service, and don't yell! Read the documentation, everything is mentioned here. https://github.com/shanteacontrols/OpenDeck/wiki/Creating-custom-board-variant

**Reply by js-midi** [reply]
I know what you mean
sorry
